install/deploy_final_non-compute_node.md

```diff
@@ -154,43 +154,43 @@ The steps in this section load hand-off data before a later procedure reboots th
             /var/log/conman \
             /var/log/zypper.log 2>/dev/null |
          sed 's_^/__' |
-         xargs tar -C / -czvf "${PITDATA}/prep/logs/pit-backup-$(date +%Y-%m-%d_%H-%M-%S).tgz"
+         xargs tar -C / -czvf "${PITDATA}/prep/logs/pit-backup-${PITBackupDateTime}.tgz"
     ```
 
-1. (`pit#`) Backup the bootstrap information from `ncn-m001`.
+1. (`pit#`) Backup the bootstrap information from `ncn-m001` before rebooting it.
 
-    > **`NOTE`** This preserves information that should always be kept together in order to fresh-install the system again.
+    > **`NOTE`** This preserves information that should always be kept together. This data can be critical for debug and support as well as in order to fresh-install the system again.
 
     1. Log in and set up passwordless SSH **to** the PIT node.
 
-        Copying **only** the public keys from `ncn-m002` and `ncn-m003` to the PIT node. **Do not** set up
+        Copying **only** the public keys from `ncn-m002` to the PIT node. **Do not** set up
         passwordless SSH **from** the PIT node or the key will have to be securely tracked or expunged if using a USB installation).
 
         > The `ssh` commands below may prompt for the NCN root password.
 
         ```bash
         ssh ncn-m002 cat /root/.ssh/id_rsa.pub >> /root/.ssh/authorized_keys &&
-             ssh ncn-m003 cat /root/.ssh/id_rsa.pub >> /root/.ssh/authorized_keys &&
              chmod 600 /root/.ssh/authorized_keys
         ```
 
-    1. Back up files from the PIT to `ncn-m002`.
+    1. Copy the prep directory to `ncn-m002` with rsync. This is required to ensure the site lan and other critical config data can be copied back to ncn-m001 after the PIT reboot.
```

Comment:
In the context of the diff I can't see why, but why does this step still exist? For the site link setup?

Maybe we can change the order to have them download the S3 backup to fetch the site link?

Otherwise this is fine if we need it.

Reply:
Technically this step will go away when the site link info is in cloudinit, and folks don't need to manually copy the pits site link info into place anymore.

Reply:
Correct. There are actually two steps that found had to restore this step. One is the site link config file and the restore of the /metal/bootstrap to m001 happens before and there are files mounted again then to use CSI before S3 access is restored. It may be possible we could configure the cray administrative access cli step before the site link is up but that seems like a significant change.